conda/recipes/versions.yaml

```
@@ -13,7 +13,7 @@ xgboost_version:
 
 # Versions for conda
 conda_version:
-  - '=4.10.3'
+  - '=4.12.0'
 conda_build_version:
   - '=3.21.7'
```

Would it be reasonable to bump `conda-build` to `3.21.8` while we are at it?

```suggestion
conda_build_version:
  - '=3.21.8'
```